infrateam_first_responder.md

```diff
@@ -2,161 +2,106 @@
-* Every team member rotates through this responsibility.
-* Encourage cross-training, since a first-responder will likely have to investigate applications with which they are unfamiliar.
-* Shore up missing and outdated documentation of production code and processes for everyone (ops, devops, PSM, stakeholders, devs, etc), informed by actual attempts to find and use said documentation to investigate production issues.
-
-### "First Responder" != "On Call"
-
-The idiosyncratic name of the role is intentional. It is not the duty of the first responder to be on-call outside of the first responder's normal work hours (which may not line up exactly with business hours in Palo Alto). The first responder rotation is an effort to watch and triage production issues in an intentional and organized fashion, since no engineer was officially assigned this responsibility in the past, and such minding of things was haphazard.
-
-# Duties
-
-## Help vet systemic puppet changes
-
-_Experiment starting July 2021_
-
-Ops will give a bit of advance notice to the infrastructure team before deploying systemic puppet changes (e.g. refactor/cleanup that touches many apps, networking changes, etc).  First responder will work with ops to determine whether testing is needed, and if so, what testing is appropriate, based on what we worry might break.  Running `infrastructure-integration-tests`  against stage after the puppet changes have been deployed there is a good default practice.  Don't hesitate to ask for help planning or performing the testing if needed.
-
-## Weekly Dependency Updates
+## Specific Responsibilities
+### Dependency Updates (once per week)
 
 The first responder needs to make sure that all codebases needing updates have updates merged and deployed. Note that some projects may need to have PRs created by hand where automatic creation may have failed. It is helpful to post updates in the `#dlss-infrastructure` Slack channel to make sure the team is aware of this work, in case anyone is working in related codebases or looking to deploy changes.
 
-### Merge 'em
+#### Merge 'em
 
 Run the `merge-all` script to automatically merge all dependency update PRs: https://github.com/sul-dlss/access-update-scripts/blob/master/merge-all.rb. Note that this script will only work with Ruby 2.6 or greater.  See the comments at the top for how to run and note you will need a github access token if you haven't previously created one.  Instructions for creating a token are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line   Save your token somewhere secure for re-use since you won't be able to view it in the Github interface again.
 
-### Deploy 'em
+#### Deploy 'em
 
 Use the `sdr-deploy` CLI, from your laptop, to deploy all infrastructure projects (with **required additional deploys** noted below) via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy.
```

Comment:
Can the parenthetical about required additional deploys be nuked since we're considering deprecating that section?

Reply:
Done.